README.md

```diff
@@ -154,6 +154,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [BetterTouchTool](http://blog.boastr.net/)
   - [BibDesk](http://bibdesk.sourceforge.net/)
   - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
+  - [Bitchx](https://github.com/BitchX/)
```

Comment:
I don't mind which one, but it should be the official one.
https://github.com/BitchX/ looks less updated than http://www.bitchx.org/
It's a mess =)